staging/src/k8s.io/client-go/dynamic/dynamicinformer/informer.go

```diff
@@ -40,25 +40,13 @@ func NewDynamicSharedInformerFactory(client dynamic.Interface, defaultResync tim
 // NewFilteredDynamicSharedInformerFactory constructs a new instance of dynamicSharedInformerFactory.
 // Listers obtained via this factory will be subject to the same filters as specified here.
 func NewFilteredDynamicSharedInformerFactory(client dynamic.Interface, defaultResync time.Duration, namespace string, tweakListOptions TweakListOptionsFunc) DynamicSharedInformerFactory {
-	return NewFilteredDynamicSharedInformerFactoryWithOptions(
```

Comment:
hmm, @ncdc do we want to undo the dynamic informer? The codegen won't do anything to make dynamic informers work with KCP

Reply:
Let's plan out what it would look like if we undid the changes to dynamic + metadata informers - could we still get kcp working?